editor.planx.uk/src/ui/editor/FlowTag/styles.ts

```diff
@@ -38,4 +38,8 @@ export const Root = styled(Box, {
   ...(tagType === FlowTagType.ServiceType && {
     backgroundColor: theme.palette.flowTag.serviceType,
   }),
+  // TODO: Is there a colour to communicate "template"?
+  ...(tagType === FlowTagType.Template && {
+    backgroundColor: theme.palette.flowTag.serviceType,
+  }),
```

Comment:
@ianjon3s I've just repeated an existing colour here, just flagging for your attention. No immediate response needed, we can pick things like this up in a final style pass on the templates feature.

Reply:
Merged for now @ianjon3s as this is feature flagged, we can come back and pick up this styling 👍